cmd/cody-gateway/internal/limiter/limiter.go

```diff
@@ -155,15 +167,15 @@ func (l StaticLimiter) TryAcquire(ctx context.Context) (_ func(context.Context,
 		}
 
 		// Set expiry on the key. If the key didn't exist prior to the previous INCR,
-		// it will set the expiry of the key to one day.
+		// it will set the expiry of the key to interval seconds.
```

Comment:
Super minor nit: since we are referring to an actual variable name, we should refer to `intervalSeconds`. (And not "interval seconds" which could be interpreted as just English text.)

```suggestion
		// it will set the expiry of the key to `intervalSeconds`.
```